thumbor/thumbor.conf

```diff
@@ -1,260 +1,221 @@
 # -*- coding: utf-8 -*-
-
-from os.path import expanduser, join
-home = expanduser("~")
-
 # thumbor imaging service
 # https://github.com/thumbor/thumbor/wiki
 
 # Licensed under the MIT license:
 # http://www.opensource.org/licenses/mit-license
 # Copyright (c) 2011 globo.com thumbor@googlegroups.com
 
-# the domains that can have their images resized
-# use an empty list for allow all sources
-#ALLOWED_SOURCES = ['mydomain.com']
+from os.path import expanduser, join
+
+from thumbor.filters import BUILTIN_FILTERS
+
+home = expanduser("~")
 
-# the max width of the resized image
-# use 0 for no max width
-# if the original image is larger than MAX_WIDTH x MAX_HEIGHT,
-# it is proportionally resized to MAX_WIDTH x MAX_HEIGHT
+## The domains that can have their images resized use an empty list for allow
+## all sources
+# ALLOWED_SOURCES = ["mydomain.com"]
+
+## The max width of the resized image # use 0 for no max width if the original
```

Comment:
Please re-read the comments you squashed. This sentence makes no sense any more (since previously they were multiple sentences).

Reply:
Agreed. Thumbor.conf is a documentation source for many users.